src/api/platform/drives/create_request.rs

```diff
@@ -8,14 +8,10 @@ use crate::api::platform::{ApiDrive, DriveKind, StorageClass};
 #[derive(Serialize)]
 pub(crate) struct CreateRequest {
     pub(crate) name: String,
-
     #[serde(rename = "type")]
     pub(crate) kind: DriveKind,
-
     pub(crate) storage_class: StorageClass,
-
-    #[serde(rename = "initial_bucket_key_pem")]
-    pub(crate) owner_key: String,
+    pub(crate) fingerprint: String,
 }
```

Comment:
Hmm without the owner_key being supplied, and without parsing banyanfs how do we initially populate the access table? We can't assume this is the API key the client is currently using as they are allowed to be different even if the common case is using the same key as the drive being created and we can't at this point know whether its a known key or not.

Reply:
this isnt actually needed, removed